elytron/src/main/java/org/wildfly/extension/elytron/ElytronDefinition.java

```diff
@@ -106,6 +108,8 @@
 
     static final StringListAttributeDefinition DISALLOWED_PROVIDERS = new StringListAttributeDefinition.Builder(ElytronDescriptionConstants.DISALLOWED_PROVIDERS)
             .setRequired(false)
+            .setAttributeParser(AttributeParser.COMMA_DELIMITED_STRING_LIST)
```

Comment:
this is wrong COMMA_DELIMITED was wrong default in the past.
it should be blank space, same as defined by xs:list.
So new attributes shouldn't use comma delimited list attributes.